kong/init.lua

```diff
@@ -1831,6 +1831,10 @@ local function serve_content(module)
 
   ngx.header["Access-Control-Allow-Origin"] = ngx.req.get_headers()["Origin"] or "*"
 
+  if kong.configuration.log_level == "debug" then
```

Comment:
I suggest cache `kong.configuration.log_level` at module level, no need to introduce 2 more table access on the proxy patch for each request.